combinations/hash.tsv

```diff
@@ -112,3 +112,4 @@ pigz=2.3.4
 jq=1.6
 r-base=4.0.3,r-shazam=1.0.2,r-tigger=1.0.0
 r-base=4.0.3,r-shazam=1.0.2,r-tigger=1.0.0,tzdata=2021a
+python=3.8.8,wget=1.20.1,biopython=1.78,igblast=1.15.0
```

Comment:
I don't think you need to pin the python vresion here. biopython should pull in python

Reply:
Ok, thanks, I was unsure if pinning it or not.